src/devices/Bmxx80/Bmx280Base.cs

```diff
@@ -271,6 +241,53 @@ protected override void SetDefaultConfiguration()
             StandbyTime = StandbyTime.Ms125;
         }
 
+        /// <summary>
+        /// Performs a temperature reading.
+        /// </summary>
+        /// <returns><see cref="Temperature"/></returns>
+        protected bool TryReadTemperatureInternal(out Temperature temperature)
```

Comment:
Our design guidelines say to name similar protected methods as `TryReadTemperatureCore`